src/others/test/lv_test_screenshot_compare.c

```diff
@@ -152,17 +115,19 @@ static bool screenshot_compare(const char * fn_ref, const char * mode, uint8_t t
     lv_refr_now(NULL);
 
     lv_draw_buf_t * draw_buf = lv_display_get_buf_active(NULL);
+    uint8_t * screen_buf_xrgb8888 = malloc(draw_buf->data_size);
```

Comment:
I get a segfault with color depth 16.

```suggestion
    uint8_t * screen_buf_xrgb8888 = malloc(draw_buf->header.w * draw_buf->header.h * 4);
```

Reply:
Oh, thanks!